src/modules/keyboardmanager/KeyboardManagerEditor/KeyboardManagerEditor.h

```diff
@@ -1,7 +1,8 @@
 #pragma once
 
 #include <KeyboardManagerState.h>
-#include <Input.h>
+#include <common/Input.h>
+#include <common/KeyboardManagerShortcuts.h>
```

Comment:
It would be better to use 
```
#include <keyboardmanager/common/Input.h>
#include <keyboardmanager/common/KeyboardManagerShortcuts.h>
```
to make a clear distinction with the `src/common` path.
And do the same in all other places where including headers from `keyboardmanager/common`